libs/blocks/aside/aside.css

```diff
@@ -46,13 +46,34 @@
 }
 
 .aside .split-image img,
-.aside .background img {
+.aside .background img,
+.aside .split-image video,
+.aside .background video {
   object-fit: cover;
   min-height: 700px;
   width: 100%;
   height: 100%;
 }
 
+.aside .background video {
+  display: block;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
```

Comment:
these are taken care of in the selector above on lines 52, 54, 55. Can remove.

Reply:
@JasonHowellSlavin I have made the suggested changes